packages/vue-components/src/Popover.vue

```diff
@@ -58,16 +69,35 @@ export default {
       isMounted: false,
     };
   },
+  computed: {
+    triggers() {
+      return this.trigger.split(' ');
```

Comment:
Might be missing the `touch` trigger here https://floating-vue.starpad.dev/guide/component.html#triggers

I'm not too sure of the implementation specifics in floating/bootstrap-vue as well but `hover` is generally clunky on mobile (https://stackoverflow.com/questions/22559756/changing-hover-to-touch-click-for-mobile-devices).
Likely, bootstrap-vue handled its specifics better. Could try investigating in this direction as well.

*ps: I'm using android, chrome

Reply:
I tried the `touch` trigger, and it displays the tooltip on "touch and hold" - when you release your finger, the tooltip disappears as well.
Should I document this behaviour in the docs?

Reply:
Ahh, thanks for trying. Not my intention (had expected it to be a substitute for `hover`).

https://github.com/MarkBind/markbind/pull/1868#discussion_r841177857